pkg/modules/prometheus/prometheus.go

```diff
@@ -119,6 +127,18 @@ func (mod *Prometheus) Validate() error {
 	return nil
 }
 
+func validateMetricsPath(path string) error {
```

Comment:
Put this logic directly in the Validate method

Reply:
@gulien, done